generator/test/error_test_data/bool_not_allowed_as_primary_key.dart

```diff
@@ -1,10 +0,0 @@
-import 'package:realm_common/realm_common.dart';
-
-//part 'bool_not_allowed_as_primary_key.g.dart';
-
-@RealmModel()
-@MapTo('Bad')
-class _Foo {
-  @PrimaryKey()
-  late bool bad;
```

Comment:
My mistake. I assumed that since bool are now indexable they could also be used for primary keys. But you are right they cannot (did a test to validate) so I have re-instituted this test.